source/Jobbr.WebApi.Tests/Jobbr.WebApi.Tests.csproj

```diff
@@ -1,133 +1,41 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
-    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
     <ProjectGuid>{CB3FE911-7034-471C-A2D7-CB89842423FC}</ProjectGuid>
-    <OutputType>Library</OutputType>
-    <AppDesignerFolder>Properties</AppDesignerFolder>
-    <RootNamespace>Jobbr.WebApi.Tests</RootNamespace>
-    <AssemblyName>Jobbr.WebApi.Tests</AssemblyName>
-    <TargetFrameworkVersion>v4.6.2</TargetFrameworkVersion>
-    <FileAlignment>512</FileAlignment>
+    <TargetFramework>net462</TargetFramework>

```

Comment:
Any reason not to target netstandard as well?

Reply:
Damn it :D toooooo many projects ;)

Reply:
Test projects have to be netcore ;) but will chagne that